OpenRA.Mods.Common/Widgets/ViewportControllerWidget.cs

```diff
@@ -118,7 +118,7 @@ public void UpdateMouseover()
 			}
 
 			var frozen = world.ScreenMap.FrozenActorsAt(world.RenderPlayer, worldRenderer.Viewport.ViewToWorldPx(Viewport.LastMousePos))
-				.Where(a => a.TooltipInfo != null && a.IsValid)
+				.Where(a => a.TooltipInfo != null && a.IsValid && (a.Actor != null && !a.Actor.IsInvisibleTo(world.RenderPlayer)))
```

Comment:
Ah - that's a very good point.

I'm at a bit of a loss as to how to solve that though.